src/Shell/GettextShell.php

```diff
@@ -260,9 +261,9 @@ private function analyzePoFile($filename): void
         $translated = $numItems - $numNotTranslated;
         $percent = 0;
         if ($numItems > 0) {
-            $percent = number_format(($translated * 100.) / $numItems, 1);
+            $percent = number_format($translated * 100. / $numItems, 1);
         }
-        $this->out(sprintf('Translated %s of items - %s %', $translated, $numItems, $percent));
+        $this->out(sprintf('Translated %s of %s items - %s %', $translated, $numItems, $percent));
```

Comment:
```suggestion
        $this->out(sprintf('Translated %s of %s items - %s %%', $translated, $numItems, $percent));
```

In `sprintf` a `%` is required to escape a `%` symbol